src/debugger/debugger.c

```diff
@@ -423,9 +450,15 @@ static int handle_breakpoints(function_stack_entry *fse, int breakpoint_type)
 
 void xdebug_debugger_handle_breakpoints(function_stack_entry *fse, int breakpoint_type)
 {
-	if (xdebug_is_debug_connection_active_for_current_pid() && XG_DBG(breakpoints_allowed)) {
-		if (!handle_breakpoints(fse, breakpoint_type)) {
-			xdebug_mark_debug_connection_not_active();
+	int block = XDEBUG_CMDLOOP_NONBLOCK;
```

Comment:
`block` here is still defined as a variable, but only used with this value in line 461. No need for this extra variable.